travis/build_info.py

```diff
@@ -0,0 +1,207 @@
+"""
+Gather Travis usage info in real-time using Travis REST APIs
+
+This only applies to travis instances that do not require
+authorization (e.g., travis-ci.org). Auth is a #TODO
+
+"""
+
+import argparse
+import logging
+import requests
+import sys
+
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = 'https://api.travis-ci.org/'
+
+
+def get_repos(org):
+    """
+    Returns list of active repos in a given org.
+    """
+
+    repo_list = []
+    # Note: One call is needed if org has <= 100 active repos.
+    # If org has > 100, this code will need to be modified
+    #  to include 'offset' requests
+    req = requests.get(
+        BASE_URL + 'v3/owner/{org}/repos?active=true'.format(org=org)
+    )
+    if req.status_code != 200:
+        raise requests.HTTPError(req.status_code)
+    req_json = req.json()
+    try:
+        for i in req_json['repositories']:
```

Comment:
I was surprised that i as a variable name got past the linter. Then checked and realized that we're not linting this repo. :disappointed: